contracts/cw-usdc/src/execute.rs

```diff
@@ -58,8 +58,12 @@ pub fn mint(
 pub fn burn(
     deps: DepsMut,
     info: MessageInfo,
+    from_address: String,
     amount: Uint128,
 ) -> Result<Response<OsmosisMsg>, ContractError> {
+    // validate that to_address is a valid address
```

Comment:
```suggestion
    // validate that from_address is a valid address
```

Reply:
Stupid question... and maybe this is just in progress
but why do we pass `from_address` and just validate it?

Reply:
This is still in progress 😄